pkg/server/controllers.go

```diff
@@ -446,29 +487,20 @@ func (s *Server) installWorkspaceScheduler(ctx context.Context, config *rest.Con
 	if err != nil {
 		return err
 	}
-
-	return s.AddPostStartHook(postStartHookName(controllerName), func(hookContext genericapiserver.PostStartHookContext) error {
-		logger := klog.FromContext(ctx).WithValues("postStartHook", postStartHookName(controllerName))
+	return s.AddPostStartHook(postStartHookName(bootstrap.ControllerNameBase), func(hookContext genericapiserver.PostStartHookContext) error {
```

Comment:
Since this is just for the "universal" CWT bootstrapper, the name and user-agent should reflect that (i.e. include "universal" somewhere)